.github/workflows/pypi-publish.yaml

```diff
@@ -0,0 +1,36 @@
+name: Build and publish Python distributions to PyPI
```

Comment:
Could you please use common workflow for publish python tarball

Reply:
Please look at the example
https://github.com/th2-net/th2-data-services/blob/dev_2.0.0/.github/workflows/publish-release.yml